src/main/java/ca/corefacility/bioinformatics/irida/ria/web/GroupsController.java

```diff
@@ -379,17 +380,24 @@ public DataTablesResponse getGroupUsers(@DataTablesRequest DataTablesParams para
 
 	/**
 	 * Get a string to tell the user which group they're going to delete.
-	 * 
-	 * @param userGroupId
-	 *            the user group that's about to be deleted.
-	 * @param model
-	 *            model for rendering view
+	 *
+	 * @param userGroupId the user group that's about to be deleted.
+	 * @param model       model for rendering view
 	 * @return a message indicating which group is going to be deleted.
 	 */
 	@RequestMapping(path = "/deleteConfirmModal", method = RequestMethod.POST)
 	public String getDeleteGroupText(final @RequestParam Long userGroupId, final Model model) {
 		final UserGroup group = userGroupService.read(userGroupId);
+		final Collection<UserGroupProjectJoin> projects = userGroupService.getProjectsWithUserGroup(group);
+		final int maxProjectsToDisplay = 3;
```

Comment:
Move this to top of the file as a global `public static final int` instead of a method scoped variable please.

Reply:
Fixed in 718fc62d032b6a56f4323020dcce64e973e34626

Reply:
Thanks for moving it. Can you change the var name to `CAPS_WITH_UNDERSCORES` so `MAX_PROJECTS_TO_DISPLAY`?  It's just a good naming convention for constant variables in Java.

Reply:
No problem. Done!  Fixed in 0502bcda79e734e6892aa9e2f7110bd7e8986625